### PR TITLE
Add isOneClickBuy attribute

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
Add isOneClickBuy attribute to the BuyComponent 

**BuyButton**: https://github.com/vtex-apps/store-components/pull/73

#### What problem is this solving?
BuyButton has the option of redirecting the user straight to checkout after being clicked.

#### How should this be manually tested?
[workspace](https://oneclickbuy--storecomponents.myvtex.com/)

#### Linked to:
[- PR 73](https://github.com/vtex-apps/store-components/pull/73)

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
```
Comment (brunojdo) [inline CHANGELOG.md]: New feature. Added

File: README.md [hunk 2]
```diff
@@ -26,15 +22,19 @@ To update the tests snapshots use
...
 ## Troubleshooting
 
 You can check if others are experiencing similar issues [here](https://github.com/vtex-apps/product-summary/issues). Also feel free to [open issues](https://github.com/vtex-apps/product-summary/issues/new).
 
-## Contributing
-
-TODO
+| Prop name          | Type       | Description                                                                 |
+| ------------------ | ---------- | --------------------------------------------------------------------------- |
+| `product`          | `Product`  | Product that owns the informations                                          |
```
Comment (brunojdo) [inline README.md]: You must do another table with the product props

File: CHANGELOG.md (hunk 1 above)
Comment (salesfelipe) [reply CHANGELOG.md]: 👍

File: README.md (hunk 2 above)
Comment (salesfelipe) [reply README.md]: 👍

Comment (salesfelipe) [reply README.md]: 👍

**Pull request merged**